canopen/network.py

```diff
@@ -25,6 +25,9 @@
 class Network(MutableMapping):
     """Representation of one CAN bus containing one or more nodes."""
 
+    NOTIFIER_CYCLE: float = 1.0  #: Maximum waiting time for one notifier iteration
+    NOTIFIER_SHUTDOWN_TIMEOUT: float = 5.0  #: Maximum waiting time to stop notifiers
```

Comment:
Make sure the docs are rendered with correct punctuation:

```suggestion
    NOTIFIER_CYCLE: float = 1.0  #: Maximum waiting time for one notifier iteration.
    NOTIFIER_SHUTDOWN_TIMEOUT: float = 5.0  #: Maximum waiting time to stop notifiers.
```

Reply:
Hm, I thought that was part of the Sphinx rendering process?  Might be confusing it with Doxygen, IIRC there the full-stops get added automatically.  Pretty annoying for me to see them in the code, actually.

Reply:
Nope, it's never been part of Sphinx AFAIK. Perhaps Doxygen, but I haven't used that in ages, so I don't remember how it behaves.

Correct punctuation in comments is a small prize to pay for nicely rendered docs, no?